### PR TITLE
Add config for travis

### PR DESCRIPTION
**Pull request by Hawatz**
### What it does
sets up continuous integration with Travis.
### DB changes (if any)
none
### UI changes (if any)
none

### CODE REVIEW

**Comment by KartowkaAteist** [comment]
Одобряю

File: Gemfile [hunk 1]
```diff
@@ -25,6 +25,12 @@ gem 'puma', '~> 3.11'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+gem 'travis'
```
Comment (alexandr-senyuk) [inline Gemfile]: lock the versions

Comment (Hawatz) [reply Gemfile]: Done

**Comment by Hawatz** [comment]
@rbUUbr @mrgrecha
Нужно ваше ревью

**Pull request merged**